app/models/scheduled_mailing.rb

```diff
@@ -1,34 +1,59 @@
 class ScheduledMailing < ApplicationRecord
+  # records an email for immediate or later delivery. Use .send_now! or .send_later! methods to schedule mails.
+
   serialize :arguments, Array
 
-  belongs_to :legal_aid_application
+  VALID_STATUSES = [
+    'waiting', # record has been written, but mail not yet sent by ScheduledMailingsDeliveryJob
```

Comment:
Might be better to remove all these comments and stuck in the readme.

Reply:
The trouble with having them in the README, as opposed to in the code is that they get out of date.  The code is updated with new statuses, but the readme isn't.